metriq_gym/job_manager.py

```diff
@@ -64,12 +67,17 @@ def _load_jobs(self):
         self.jobs = []
         if os.path.exists(self.jobs_file):
             with open(self.jobs_file) as file:
-                for line in file:
+                for line_no, line in enumerate(file, start=1):
                     try:
                         job = MetriqGymJob.deserialize(line.strip())
                         self.jobs.append(job)
-                    except json.JSONDecodeError:
-                        continue
+                    except Exception as exc:  # noqa: BLE001
```

Comment:
Particularly for this use case, it's important to be explicit about the exception on which we are lenient (instead of ignoring the ruff violation, that is)